source/docs/api-reference/api-usage/status-signals.rst

```diff
@@ -181,6 +182,30 @@ The following signals are time-synchronized:
 ---------------------
 
 All ``StatusSignalValue`` objects have a ``getDataCopy()`` method that returns a new ``SignalMeasurement`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/StatusSignalValue.SignalMeasurement.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/structctre_1_1phoenixpro_1_1_signal_measurement.html>`__) object.
+
 ``SignalMeasurement`` is a `Passive Data Structure <https://en.wikipedia.org/wiki/Passive_data_structure>`__ that provides all the information about a signal at the time of the ``getDataCopy()`` call, which can be useful for data logging.
 
 .. warning:: ``getDataCopy()`` returns a **new** ``SignalMeasurement`` object every call. **Java** users should **avoid** using this API in RAM-constrained applications.
+
+Latency Compensation
+--------------------
+
+Phoenix Pro provides built-in support for latency compensation via a static function ``getLatencyCompensation()`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/BaseStatusSignalValue.html#getLatencyCompensatedValue(com.ctre.phoenixpro.StatusSignalValue,com.ctre.phoenixpro.StatusSignalValue)>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1_base_status_signal_value.html#a22f020db5abbf556ac7605024309bb26>`__). This function takes two arguments: **signal** and **signalSlope**. **signal** represents the signal to be compensated and **signalSlope** is the rate of change of the signal.
```

Comment:
"Users can perform latency compensation using \`\`BaseStatusSignalValue.getLatencyCompensatedValue()\`\` (API doc links)."

I don't think the rest of the paragraph is necessary, as it's in the API docs.